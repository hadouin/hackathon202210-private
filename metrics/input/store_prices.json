{
	"1": {
		"type": "clone_product",
		"arguments": {"product_id": 1, "new_product_id": 100, "coef": 2}
	},
	"2": {
		"type": "sum_of_prices",
		"arguments": {"product_id": 100}
	},
	"3": {
		"type": "clone_product",
		"arguments": {"product_id": 2, "new_product_id": 100, "coef": 5}
	},
	"5": {
		"type": "sum_of_prices",
		"arguments": {"product_id": 100}
	},
	"6": {
		"type": "clone_product",
		"arguments": {"product_id": 2, "new_product_id": 100, "coef": 3}
	},
	"7": {
		"type": "sum_of_prices",
		"arguments": {"product_id": 100}
	},
	"8": {
		"type": "clone_product",
		"arguments": {"product_id": 3, "new_product_id": 100, "coef": 0.2}
	},
	"9": {
		"type": "sum_of_prices",
		"arguments": {"product_id": 100}
	},
	"10": {
		"type": "clone_product",
		"arguments": {"product_id": 1, "new_product_id": 100, "coef": 0.5}
	},
	"11": {
		"type": "sum_of_prices",
		"arguments": {"product_id": 100}
	},
	"12": {
		"type": "clone_product",
		"arguments": {"product_id": 3, "new_product_id": 100, "coef": 5}
	},
	"13": {
		"type": "sum_of_prices",
		"arguments": {"product_id": 100}
	},
	"14": {
		"type": "clone_product",
		"arguments": {"product_id": 3, "new_product_id": 100, "coef": 5}
	},
	"15": {
		"type": "sum_of_prices",
		"arguments": {"product_id": 100}
	},
	"16": {
		"type": "delete_product",
		"arguments": {"product_id": 100}
	}
}
