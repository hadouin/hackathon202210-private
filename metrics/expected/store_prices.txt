1 1000
2 100410440
3 1000
5 348596340
6 1000
7 497507880
8 1000
9 507435979
10 1000
11 532538589
12 1000
13 780741069
14 1000
15 1028943549
16 Product 100 was successefully deleted
